torchci/lib/bot/README.md

```diff
@@ -2,28 +2,20 @@
 
 - [PyTorch Bot Architecture Analysis](#pytorch-bot-architecture-analysis)
   - [Overview](#overview)
-  - [Core Architecture](#core-architecture)
-    - [Entry Points](#entry-points)
-    - [Command System](#command-system)
-    - [Permission System (`lib/bot/utils.ts:248`)](#permission-system-libbotutilsts248)
-  - [Bot Modules](#bot-modules)
-    - [Core Command Bots](#core-command-bots)
-    - [Automation Bots](#automation-bots)
-    - [CI Integration Bots](#ci-integration-bots)
-    - [Security \& Review Bots](#security--review-bots)
-    - [Infrastructure Bots](#infrastructure-bots)
-  - [Detailed Bot Analysis](#detailed-bot-analysis)
-    - [1. autoLabelBot.ts](#1-autolabelbotts)
-    - [2. autoCcBot.ts](#2-autoccbotts)
-    - [3. retryBot.ts](#3-retrybotts)
-    - [4. ciflowPushTrigger.ts](#4-ciflowpushtriggerts)
+  - [Core Bots](#core-bots)
+    - [pytorchBot](#pytorchbot)
+    - [ciflowPushTrigger.ts](#ciflowpushtriggerts)
       - [Configuration (ciflow_push_tags)](#configuration-ciflow_push_tags)
-    - [5. cancelWorkflowsOnCloseBot.ts](#5-cancelworkflowsonclosebotts)
-    - [6. verifyDisableTestIssueBot.ts](#6-verifydisabletestissuebotts)
-    - [7. stripApprovalBot.ts](#7-stripapprovalbotts)
-    - [8. codevNoWritePermBot.ts](#8-codevnowritepermbotts)
-    - [9. drciBot.ts](#9-drcibotts)
-    - [10. webhookToDynamo.ts](#10-webhooktodynamots)
+    - [webhookToDynamo.ts](#webhooktodynamots)
```

Comment:
what's the definition of the "Core bots".  should we preserve the distinction between bots that users interact with (pytorchbot, ciflow) and purely infrastructure ones (webhookToDynamo)?

Reply:
Removed core + other, just one bot modules section

I was hoping to have a vague ordering to the bots from most -> least important, both from a infra and a user perspective, but it's hard to define what's important so I'll just get rid of it